### PR TITLE
update octocats landingpage for aurae

### PR DESCRIPTION
**Pull request by LastChain**
Further updates on README.md 
(1st iteration)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,91 @@
...
+<!-- THE DOCUMENT -->
+
+![Deploy (aurae.io)] ![Documentation] ![Main Build (lint, compile, test)]
+
+# Summary
+
+[Aurae] is a free and [open-source] project built with the [Rust] systems
+language.
+
+The project adapts to the expanding demands of enterprise distributed systems
```
Comment (krisnova) [inline README.md:10]: I have very opinionated that the first sentence of the repo should either answer very concisely either:

 1. What the tool IS.
 2. What the tool DOES. 

In the case of Aurae I would want the first sentence to answer what the tool IS. 

> Aurae is a memory safe runtime daemon, process manager, and init pid 1 system designed to remotely schedule processes, containers, virtual machines, and configure node configuration such as networking an storage.

File: README.md [hunk 2]
```diff
@@ -1,67 +1,91 @@
...
+proportions by utilizing a variety of isolation strategies for enterprise
+workloads, the project can work exceptionally well with tools like Kubernetes [^compare].
+
+## Project Status
+
+> **STILL IN EARLY DEVELOPMENT!**<br>
+> **The Aurae project and API can change without notice.**<br> 
+> **Do not run the project in production until further notice!**
+> <br><br>
+> _"Aurae attempts to simplify and improve the stack in_
```
Comment (krisnova) [inline README.md:25]: I suggest getting rid of my quote here. I don't need to be (nor do I want to be) the "star of the show". We can have the quote, just maybe get it off the main repo.

File: README.md [hunk 3]
```diff
@@ -1,67 +1,91 @@
...
+- The Aurae project welcomes contributions of all kinds and sizes.
+- Please read the "[getting involved]" documentation before contributing to the
+  project.
+- You do not have to know [Rust] to join the project.
+
+By joining the project in its early phases, you'll participate in the
+development of a milestone candidate for enterprise distributed systems and
+automation.
+
+# **Aurae**
```
Comment (krisnova) [inline README.md:43]: I don't understand this section? Is this just "more detail" or "expanded overview"? If so we should call it that. There isn't actually a binary or tool called "aurae" so maybe we shouldn't use it as a nebulous term/title?

**Comment by LastChain** [comment]
Thank you for the feedback. 
For further reference should I have used the Git-Drafts function for this PR?

File: README.md (hunk 3 above)
Comment (LastChain) [reply README.md]: This is helpful. Changed it accordingly.

File: README.md (hunk 2 above)
Comment (LastChain) [reply README.md]: Pardon me. For that one.

File: README.md (hunk 1 above)
Comment (LastChain) [reply README.md]: Noted and applied accordingly.

File: README.md [hunk 4]
```diff
@@ -1,67 +1,78 @@
...
+[Aurae] is a memory-safe runtime daemon, process manager, and PID-1 initialization
+system designed to remotely schedule processes, containers, and virtual machines
+as well as set node configurations like networking storage.
 
-Aurae is on a mission to be the most loved and effective way of managing workloads on a single piece of hardware. Our hope is that by bringing a better set of controls to a node, we can unlock brilliant higher order distributed systems in the future.
 
-Aurae takes ownership of all runtime processes on a single piece of hardware, and provides mTLS encrypted gRPC APIs ([Aurae Standard Library](https://aurae.io/stdlib/)) to manage the processes. With Aurae [Cells](https://aurae.io/blog/24-10-2022-aurae-cells/) the project offers a way to slice up a system using various isolation strategies for enterprise workloads.
+The [open-source] project adapts to the expanding demands of enterprise
+distributed systems and meets the present and future demands for brilliant
+higher-order distributed systems using the [Rust] systems language.
```
Comment (future-highway) [inline README.md:18]: This makes it seem like higher-order distributed systems would also need to be written in rust.

Also, while "brilliant" worked with the original wording, I don't think it does anymore.

Comment (LastChain) [reply README.md:18]: Thank you for taking the time I highly valued your intput.

Will change it accordingly.

Comment (LastChain) [reply README.md]: > This makes it seem like higher-order distributed systems would also need to be written in rust.
> 
> Also, while "brilliant" worked with the original wording, I don't think it does anymore.

Adapted and applied.

File: README.md [hunk 5]
```diff
@@ -1,67 +1,78 @@
...
 
-The `auraed` daemon can be run as a PID 1 on a Linux kernel and manages containers, virtual machines, and spawning short-lived nested virtual instances of itself for an additional layer of isolation.
+# Summary
 
-### Mission
+[Aurae] is a memory-safe runtime daemon, process manager, and PID-1 initialization
+system designed to remotely schedule processes, containers, and virtual machines
+as well as set node configurations like networking storage.
 
-Aurae is on a mission to be the most loved and effective way of managing workloads on a single piece of hardware. Our hope is that by bringing a better set of controls to a node, we can unlock brilliant higher order distributed systems in the future.
```
Comment (krisnova) [inline README.md:11]: The mission statement is important as the project is still so early. People need a way to understand the motivation and our vision quickly. 

Can we please keep this language:

> Aurae is on a mission to be the most loved and effective way of managing workloads on a node. Our hope is that by bringing a better set of controls to a node, we can unlock brilliant higher order distributed systems in the future.

Comment (LastChain) [reply README.md:11]: I will implement it exactly as is.

File: README.md [hunk 6]
```diff
@@ -1,67 +1,91 @@
...
+> **Do not run the project in production until further notice!**
+> <br><br>
+> _"Aurae attempts to simplify and improve the stack in_
+> _enterprise distributed systems by carving out a small portion of_
+> _responsibility while offering a few basic guarantees with regard to state,_
+> _synchronicity, awareness, and security."_
+> <br>&mdash; [Kris Nóva] (Founder of the Nivenly Foundation)
+
+_Auraes mission is to become the most loved method of managing workloads on a_
+_single piece of hardware._
```
[the remark this answers is not in the document]
Comment (LastChain) [reply README.md]: corrected.

File: README.md [hunk 7]
```diff
@@ -1,67 +1,74 @@
...
 
-Aurae takes ownership of all runtime processes on a single piece of hardware, and provides mTLS encrypted gRPC APIs ([Aurae Standard Library](https://aurae.io/stdlib/)) to manage the processes. With Aurae [Cells](https://aurae.io/blog/24-10-2022-aurae-cells/) the project offers a way to slice up a system using various isolation strategies for enterprise workloads.
+[Aurae] deploys a memory-safe [^memory-safe] runtime daemon, process manager,
+and PID-1 initialization system to remotely schedule processes, containers, and
+virtual machines as well as set node configurations (e.g., like networking
+storage).
 
-### Project Status
+Through system proportioning and enterprise workload isolation techniques, the
+Aurae [open-source] project can complement Kubernetes, as Aurae supports the
```
Comment (krisnova) [inline README.md:23]:
```suggestion
Aurae [open-source] project can complement higher order schedulers and control planes (such as Kubernetes) as Aurae supports the
```

**Review by krisnova — APPROVED** [review]

**Pull request merged**